.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
+Pipfile.lock
```

Comment:
Lock files should always be committed to version history: https://developerexperience.io/practices/lockfile#:~:text=A%20lock%20file%20contains%20important,from%20the%20same%20dependency%20definition.

Can you remove this from .gitignore?